### DISCUSSION TITLE
Is there a getting started guide for plugin development?

### DISCUSSION THREAD

**Discussion by punkscience** [audacious-media-player/audacious#1380] [Q&A]
Hey folks - I would like to add support to the scrobble plugin for scrobbling to Listenbrainz in addition to Last.fm. Is there a guide to how to set up the workflow to get started and how the plugins are setup etc?

**Comment by jlindgren90** [comment]
Unfortunately, there isn't a "getting started" guide. You'll have to look at the code of existing plugins, and check the documentation that exists in the libaudcore headers.
